src/components/metadata/styles/metadata.scss

```diff
@@ -24,9 +24,17 @@
   padding: 6px 0 24px 0;
   overflow-y: auto;
   background: var(--color-metadata-bg);
+
+  // Subtle shadow when overlapping sidebar
+  box-shadow: 0 0 18px rgba(0, 0, 0, 0.04);
```

Comment:
1. This should be different for light and dark themes.
2. This element already has a themed box-shadow applied (via `@extend %sidebar;`), which this rule is overriding. Is this shadow better than the default one?
  ![image](https://user-images.githubusercontent.com/1155816/104356395-d0b94e00-5503-11eb-9a48-83d1ad51aa01.png)

Reply:
note: just realised that the `%sidebar` rule is broken. will fix it.

Reply:
update: I fixed this in a separate PR which was merged to main yesterday

Reply:
Ah thanks I thought it had been removed, I'll remove this one then.